io/pci/distro_tools.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Author: Bimurti Bidhibrata Pattjoshi <bbidhibr@in.ibm.com>
+#
+
+"""
+Tests the different tool
+"""
+
+from avocado import main
+from avocado import Test
+from avocado.utils import process
+from avocado import skipUnless
+
+IS_POWER_VM = 'pSeries' in open('/proc/cpuinfo', 'r').read()
+
+
+class DisrtoTool(Test):
+    '''
+    to test different type of tool
+    '''
+    @skipUnless(IS_POWER_VM,
+                "supported only on PowerVM platform")
+    def setUp(self):
+        '''
+        to check host platform
+        '''
+        self.option = self.params.get("option", default='')
+
+    def test(self):
+        '''
+        test the lsslot tool
+        '''
+        cmd = "lsslot"
+        if self.option == "pci":
+            cmd = "%s -d %s" % (cmd, self.option)
```

Comment:
Why is there a failure in job.log in this option ?

Reply:
now its fine, before its was lsslot -d so that command is not working,later i saw that is lsslot -d pci.